tests/aws/services/events/test_events_integrations.py

```diff
@@ -237,6 +237,251 @@ def test_put_events_with_target_lambda(create_lambda_function, cleanups, aws_cli
     assert actual_event["detail"] == EVENT_DETAIL
 
 
+@markers.aws.validated
+@pytest.mark.skipif(is_v2_provider(), reason="V2 provider does not support this feature yet")
+def test_put_events_with_target_lambda_list_entry(
+    create_lambda_function, cleanups, aws_client, clean_up, snapshot
+):
+    rule_name = f"rule-{short_uid()}"
+    function_name = f"lambda-func-{short_uid()}"
+    target_id = f"target-{short_uid()}"
+    bus_name = f"bus-{short_uid()}"
+
+    # clean up
+    cleanups.append(lambda: aws_client.lambda_.delete_function(FunctionName=function_name))
+    cleanups.append(lambda: clean_up(bus_name=bus_name, rule_name=rule_name, target_ids=target_id))
+
+    rs = create_lambda_function(
+        handler_file=TEST_LAMBDA_PYTHON_ECHO,
+        func_name=function_name,
+        runtime=Runtime.python3_9,
+    )
+
+    func_arn = rs["CreateFunctionResponse"]["FunctionArn"]
+
+    event_pattern = {"detail": {"payload": {"automations": {"id": [{"exists": True}]}}}}
+
+    aws_client.events.create_event_bus(Name=bus_name)
+    rs = aws_client.events.put_rule(
+        Name=rule_name,
+        EventBusName=bus_name,
+        EventPattern=json.dumps(event_pattern),
+    )
+    aws_client.lambda_.add_permission(
+        FunctionName=function_name,
+        StatementId=f"{rule_name}-Event",
+        Action="lambda:InvokeFunction",
+        Principal="events.amazonaws.com",
+        SourceArn=rs["RuleArn"],
+    )
+    rs = aws_client.events.put_targets(
+        Rule=rule_name,
+        EventBusName=bus_name,
+        Targets=[{"Id": target_id, "Arn": func_arn}],
+    )
+
+    assert "FailedEntryCount" in rs
+    assert "FailedEntries" in rs
+    assert rs["FailedEntryCount"] == 0
+    assert rs["FailedEntries"] == []
+
+    event_detail = {
```

Comment:
@maxhoheiser Moving forward with EventBridge testing, we could consider externalizing long JSON events from the Python codebase (similar to https://github.com/localstack/localstack/pull/10599) to benefit from better JSON syntax highlighting and declutter the test codebase.